pages/upgrading_an_app.md

```diff
@@ -35,30 +35,31 @@ For this sub-generator to work you need to have `git` installed from [http://git
 
 Go into the application's root directory:
 
-`cd myapplication/`
+`cd myJHipsterProject/`
 
 To upgrade your application, type:
 
 `npx jhipster upgrade`
```

Comment:
Not sure if this is still valid from jh8 and up, as we have extracted the update/migrate into a dedicated blueprint. https://github.com/mshima/generator-jhipster-migrate

Maybe @mraible can tell more as I have never used the upgrade mechanism.

Reply:
Is this just intended to be a simple prompt that you can't perform an upgrade from an unrelated project folder? I only want to make it more explicit for semantics, `myapplication` ... which one of _my applications_?